homeassistant/helpers/__init__.py

```diff
@@ -98,14 +98,17 @@ def config_per_platform(config, domain, logger):
 
     while config_key in config:
```

Comment:
Yeah you're right. Let's keep it your approach. That way we can eventually phase the old way out.